test/benchmark-common-tasks/main.cpp

```diff
@@ -1630,6 +1630,72 @@ struct BenchmarkSort : BenchmarkWithStrings {
     }
 };
 
+// benchmark for testing compressed string sorting.
+// N is the size of the string to generate
+// M is the numer of times we want store the string (number of dups)
+template <size_t N, size_t M>
+struct BenchmarkSortCompressed : BenchmarkWithStrings {
+    mutable std::string _name;
+
+    BenchmarkSortCompressed()
+        : BenchmarkWithStrings()
+    {
+        if constexpr (N <= 15) {
+            _name = util::format("SortCompressedSmall(%1,%2)", std::to_string(N), std::to_string(M));
+        }
+        else if constexpr (N <= 63) {
+            _name = util::format("SortCompressedMedium(%1,%2)", std::to_string(N), std::to_string(M));
+        }
+        else {
+            _name = util::format("SortCompressedLarge(%1,%2)", std::to_string(N), std::to_string(M));
+        }
+    }
+
+    const char* name() const
+    {
+        return _name.c_str();
+    }
+
+    void before_all(DBRef group)
+    {
+        BenchmarkWithStringsTable::before_all(group);
```

Comment:
this adds a bunch of strings to the table, do you really want those in there?

Reply:
no.. removed.